content/ops/secrets.md

```diff
@@ -0,0 +1,104 @@
+---
+menu:
+  main:
+    parent: operations
+title: Secrets Management
+weight: 10
+---
+
+To meet NIST security control [SC-12 (1)](https://web.nvd.nist.gov/view/800-53/Rev4/control?controlName=SC-12) we maintain the availability of all information on the platform in the event a crypotographic access key is lost or compromised.
+
+Authorized federal staff rotate, encrypt, and backup keys monthly. Privileged users can access the keys only with two-factor authentication and a decryption passphrase. In the rare case that both the keys and the decryption passphrase for the backup are lost or compromised, new keys can be rotated in by authorized federal staff, while maintaining availability of information.
+
+### Grabbing AWS credentials
+
+Note that access here is gated by membership in [the cloud-gov-ci team](https://github.com/orgs/18F/teams/cloud-gov-ci) in GitHub.
+
+1. Go to https://ci.fr.cloud.gov/pipelines/terraform-provision (substitute relevant Concourse instance).
+1. Click the `bootstrap-tooling` job.
+1. Expand the `create-update-tooling` step.
+1. Grab `ci_access_key_id` and `ci_secret_access_key`.
```

Comment:
Would love to be able to get rid of this section --> https://github.com/18F/cg-atlas/issues/70